operations/configuration.md

```diff
@@ -393,31 +393,33 @@ Returns all tax environments supported by the API.
 | `Value` | number | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
 | `Strategy` | [Tax Rate Strategy](#tax-rate-strategy) | required | Tax strategy type, e.g. relative or flat. |
 
-#### Tax Rate Strategy
+#### Tax rate strategy
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `Discriminator` | [Tax Strategy Discriminator](#tax-strategy-discriminator) | required | If tax rate is flat or relative. |
-| `Value` | object | required | Based on Tax Strategy Discriminator, e.g. [Relative Rate Strategy](#relative-rate-strategy) or [Flat Rate Strategy](#flat-rate-strategy). |
+| `Value` | object | required | Structure of the object depends on [Tax Strategy Discriminator](#tax-strategy-discriminator-data). |
 
-#### Flat Rate Strategy
+#### Tax strategy discriminator
+
+* `Flat` - Used when relevant tax rate is flat (e.g. 5.00 EUR). 
+* `Relative` - Used when relevant tax rate is relative (e.g. 21%)
```

Comment:
```suggestion
* `Flat` - Used when relevant tax rate is flat (e.g. 5.00 EUR) ([Flat tax rate strategy data](#flat-tax-rate strategy data)) 
* `Relative` - Used when relevant tax rate is relative (e.g. 21%) ([Relative tax rate strategy data](#relative-tax-rate strategy data))
```

Reply:
Having two parentheses, one after each other is ugly. Also sentences should end with dots.

Reply:
I hope that the developers using this documentation are intelligent enough to determine which of the two they need without having yet another link so let's just stick with the example in the parentheses here.